spec/graphql/relay/array_connection_spec.rb

```diff
@@ -157,13 +178,13 @@ def get_page_info(result)
 
       it "applies to queries by `last`" do
         last_cursor = "Ng=="
-        second_to_last_two_names = ["Death Star", "Shield Generator"]
+
         result = star_wars_query(query_string, "last" => 100, "before" => last_cursor)
-        assert_equal(second_to_last_two_names, get_names(result))
+        assert_equal(["Death Star", "Shield Generator"], get_names(result))
         assert_equal(true, get_page_info(result)["hasPreviousPage"])
 
         result = star_wars_query(query_string, "before" => last_cursor)
-        assert_equal(second_to_last_two_names, get_names(result))
+        assert_equal(["Yavin", "Echo Base"], get_names(result))
```

Comment:
Yes, you're right.

Previously, using `before` implied `last: max_page_size` automatically (unless you specified `first`). This is not according to the relay connection spec, which specifies that `before` and `after` are not tied to `first` and `last` in any way.

This might be a breaking change to users, but I'm not sure. 🤔  The previous behaviour was confusing and wrong, so I hope people don't rely on it.